packages/fxa-settings/src/models/reliers/options/browser-relier-options.ts

```diff
@@ -0,0 +1,8 @@
+import { RelierOptions } from "./relier-options";
+
+/** Options specific to browser relier */
+
+export interface BrowserRelierOptions extends RelierOptions {
+  // Deprecated... used by VAT, and we are moving away from this
+  translator?: any | undefined;
```

Comment:
Hmm, maybe reference this (or make reference here) in whatever follow up ticket we make since yeah we should be able to nix this.